### PR TITLE
Do not send projectId on configuration endpoint

### PR DESCRIPTION
**Pull request by onhate**
That's been enforced by the endpoint now

### CODE REVIEW

File: node/env0-deploy-utils.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ class DeployUtils {
     }
 
     console.log(`setting the following configuration: ${JSON.stringify(configuration)}`);
-    await apiClient.callApi('post', 'configuration', {data: configuration});
+    await apiClient.callApi('post', 'configuration', {data: {...configuration, projectId: undefined}});
```
Comment (omryhay1) [inline node/env0-deploy-utils.js]: bitmoji

@onhate - I know that I wrote the 99% of the code here, and it's my fault, but @avnerenv0 started writing unit tests for this project, so let's try to add a specific unit test for each change as we go along with this project.

Comment (onhate) [reply node/env0-deploy-utils.js]: done :)

Comment (onhate) [reply node/env0-deploy-utils.js]: we probably need to configure github actions to run the tests now

Comment (onhate) [reply node/env0-deploy-utils.js]: https://github.com/env0/env0-client-integrations/pull/9/checks?check_run_id=387570780

**Pull request merged**